docs/Installation.md

```diff
@@ -499,35 +504,22 @@ sudo systemctl enable mariadb
 sudo systemctl start mariadb
 ```
 
-To initialize the database (unless you keep an old database) connect to the
-MariaDB server:
+To create the database and the database user (unless you keep an old database)
+run the command. Edit the script first if you want a non-default user name or
+password.
 
 ```sh
-sudo mysql
-```
-
-Create the database:
-```sql
-CREATE DATABASE zonemaster;
+sudo mysql -e "CREATE DATABASE zonemaster;" -e "CREATE USER 'zonemaster'@'localhost' IDENTIFIED BY 'zonemaster';" -e "GRANT ALL ON zonemaster.* TO 'zonemaster'@'localhost';"

```

Comment:
Would it be possible to split the command in 3 commands? Something like:
```
sudo mysql -e "CREATE DATABASE zonemaster;"
sudo mysql -e "CREATE USER 'zonemaster'@'localhost' IDENTIFIED BY 'zonemaster';"
sudo mysql -e "GRANT ALL ON zonemaster.* TO 'zonemaster'@'localhost';"
```

Reply:
Does that work well if you paste all three commands at the same time? How till the mysql prompting of the password be?

Reply:
@pnax, I tested your model on Ubuntu. I pasted all three lines (commands) at once. It works well if the sudo authentication is cached, i.e. you have just run another command with sudo, but it does not work well if you the cache is clean, e.g. just logged in. I think it is better to stick with one command.

Reply:
> It works well if the sudo authentication is cached, i.e. you have just run another command with sudo, but it does not work well if you the cache is clean, e.g. just logged in. I think it is better to stick with one command.

Well regarding the rest of the document, it does not seem to be inconsistent to have several commands with sudo in the same block (see the block using `install` and the one enabling and starting the services).

Reply:
The other blocks can just be repeated to get the correct result. That is not possible with the database creation step. I agree that there are pros with shorter lines, but should we risk that one can get stuck?

Reply:
> The other blocks can just be repeated to get the correct result. That is not possible with the database creation step.

I've just tried repeating the commands and it works. I don't see how it's not possible to do that here.

> should we risk that one can get stuck?

I don't see how you are stuck and can't continue the installation. I understand that if you have not run a command with sudo before, copy/pasting might result in a prompt to gain sudo privileges but this will be the case any way. So I don't get what is blocking here.

Reply:
@pnax, in my environment it does not work. I get prompted twice by sudo. I do not enter anything in between. Sudo eats the the second and the third lines. If you look carefully you will see, but then it is too late and you have to remove the database.

```
$ sudo mysql -e "CREATE DATABASE zonemaster;" 
[sudo] password for matsd: 
Sorry, try again.
[sudo] password for matsd: 
$ sudo -u zonemaster $(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')/create_db.pl
DBI connect('database=zonemaster;host=localhost;port=3306','zonemaster',...) failed: Access denied for user 'zonemaster'@'localhost' at /usr/local/share/perl/5.26.1/Zonemaster/Backend/DB.pm line 377.
```